conda.rb

```diff
@@ -47,8 +47,8 @@ def update_packages
       @redis.sadd("package_names", packages.keys.map { |name| "#{channel}/#{name}" })
       packages.each do |name, package_info|
         version = package_info["version"]
-        key = "#{channel}/#{name}"
-        @redis.set("packages:#{key}", MessagePack.pack(version))
+        key = "packages:#{channel}/#{name}"
+        @redis.set(key, MessagePack.pack(package_info))
```

Comment:
Glad it was this easy.
cocoapods-api's "pod_info" must have a key of `version` that was an object, not just a string. so this was only storing string. 
Thanks Mike